feat: translate time in 3.13

ken71301 opened the pull request:
- rst:974，需釐清 if one is defined 的 one 為何
- rst:1017 原文令人疑惑

ken71301 committed:
feat: translate time in 3.13
- rst:974，需釐清 if one is defined 的 one 為何
- rst:1017 原文令人疑惑

ken71301 committed:
fix: render error

ken71301 committed:
fix: render error

ken71301 committed:
fix: render error

mattwang44 commented on library/time.po:
```diff
@@ -642,6 +646,8 @@ msgid ""
 "Raises an :ref:`auditing event <auditing>` ``time.sleep`` with argument "
 "``secs``."
 msgstr ""
+"引發一個帶有引數 ``secs`` 的\\ :ref:`審計事件 (auditing event) <auditing>` "
```
之前是都翻「稽核事件」
```suggestion
"引發一個帶有引數 ``secs`` 的\\ :ref:`稽核事件 (auditing event) <auditing>` "
```

mattwang44 commented on library/time.po:
```diff
@@ -662,7 +668,7 @@ msgstr ""
 #: ../../library/time.rst:413
 msgid "Raises an auditing event."
-msgstr ""
+msgstr "引發一個審計事件。"
```
ditto
```suggestion
msgstr "引發一個稽核事件。"
```

mattwang44 commented on library/time.po:
```diff
@@ -1266,24 +1272,28 @@ msgid ""
 "of two calls in the same thread is valid."
 msgstr ""
+"回傳當前執行緒的系統和用戶 CPU 時間之和（以帶有小數的秒數表示）。它不包括睡眠"
+"期間經過的時間。根據定義，這是執行緒相關 (thread-specific) 的。回傳值的參照點"
```
nit: 沒想到比較通暢的翻譯，但 specific 跟「相關」好像不太一樣
```suggestion
"期間經過的時間。根據定義，這是執行緒特定 (thread-specific) 的。回傳值的參照點"
```

mattwang44 commented on library/time.po:
```diff
@@ -1266,24 +1272,28 @@ msgid ""
 #: ../../library/time.rst:723
 msgid "Unix systems supporting ``CLOCK_THREAD_CPUTIME_ID``."
-msgstr ""
+msgstr "Unix 系統支援 ``CLOCK_THREAD_CPUTIME_ID`` 。"
```
放在 `Availability` 底下應該是補充說明

```suggestion
msgstr "有支援 ``CLOCK_THREAD_CPUTIME_ID`` 的 Unix 系統。"
```

mattwang44 commented on library/time.po:
```diff
@@ -1329,7 +1348,7 @@ msgstr "``std`` 和 ``dst``"
 "Three or more alphanumerics giving the timezone abbreviations. These will be "
 "propagated into time.tzname"
-msgstr ""
+msgstr "三個或更多字母數字組成的時區縮寫。這些縮寫會被傳播到 time.tzname 中。"
```
nit
```suggestion
msgstr "三個或更多字母與數字 (alphanumerics) 組成的時區縮寫。這些縮寫會被傳播到 time.tzname 中。"
```

mattwang44 commented on library/time.po:
```diff
@@ -1384,12 +1410,17 @@ msgid ""
 "first week in which the *d*'th day occurs. Day zero is a Sunday."
 msgstr ""
+"一年中第 *m* 月的第 *n* 週的第 *d* 天（0 <= *d* <= 6，1 <= *n* <= 5，1 <= "
+"*m* <= 12，其中第 5 週表示“該月的最後一個 *d* 天”，這可能出現在第四或第五"
```
```suggestion
"*m* <= 12，其中 *n* 為 5 表示「該月的最後一個第 *d* 天」，這可能出現在第四或第五"
```

mattwang44 commented on library/time.po:
```diff
@@ -1421,6 +1452,12 @@ msgid ""
 "``'Australia/Melbourne'``, ``'Egypt'`` or  ``'Europe/Amsterdam'``. ::"
 msgstr ""
+"在許多 Unix 系統（包括 \\*BSD、Linux、Solaris 和 Darwin）上，使用系統的 "
+"zoneinfo (\\ :manpage:`tzfile(5)`\\ ) 資料庫來指定時區規則會更加方便。要這樣"
```
nit
```suggestion
"zoneinfo (:manpage:`tzfile(5)`) 資料庫來指定時區規則會更加方便。要這樣"
```

mattwang44 commented on library/time.po:
```diff
@@ -1421,6 +1452,12 @@ msgid ""
+"做，請將 :envvar:`TZ` 環境變數設定為所需時區資料檔案的路徑，相對於系統 "
+"'zoneinfo' 時區資料庫的根目錄，通常位於 :file:`/usr/share/zoneinfo`。例如，"
+"``'US/Eastern'``、``'Australia/Melbourne'``、``'Egypt'`` 或 ``'Europe/"
+"Amsterdam'``。： ::"
```
```suggestion
"Amsterdam'``。 ::"
```

mattwang44 commented on library/time.po:
```diff
@@ -1457,6 +1503,9 @@ msgid ""
 "have  discontinuities if the time is changed using ``settimeofday()`` or "
 "similar."
 msgstr ""
+"這允許應用程序獲取一個能夠感知暫停的單調時鐘，而無需處理 :data:"
```
```suggestion
"這允許應用程式獲取一個能夠感知暫停的單調時鐘，而無需處理 :data:"
```

mattwang44 commented on library/time.po:
```diff
@@ -1513,26 +1568,31 @@ msgid ""
 "suspended, providing accurate uptime measurement, both absolute and interval."
 msgstr ""
+"表示系統運作且無暫停的絕對時間，提供包括絕對時間 (absolute) 和時間區間 "
+"(interval) 的精確的正常運行時間 (uptime) 測量。"
```
nit: uptime 翻成「上線時間」？

ken71301 replied:
ok

mattwang44 reviewed (commented):
> rst:974，需釐清 if one is defined 的 one 為何
> rst:1017 原文令人疑惑

我覺得都沒問題，要再幫忙 rebase 一下

ken71301 commented on library/time.po:
```diff
@@ -1481,86 +1528,104 @@ msgid ""
 "suspended, providing accurate uptime measurement, both absolute and interval."
 msgstr ""
+"表示系統運作且無暫停的絕對時間，提供包括絕對時間 (absolute) 和時間區間 "
+"(interval) 的精確的正常運行時間 (uptime) 測量。"
```
```suggestion
"(interval) 的精確的正常上線時間 (uptime) 測量。"
```

ken71301 committed:
Apply suggestions from code review

Co-authored-by: Wei-Hsiang (Matt) Wang <mattwang44@gmail.com>

ken71301 committed:
feat(time.po): fill some missed translation

ken71301 committed:
fix: try to fix inconsistent term references in translated message

/home/runner/work/python-docs-zh-tw/cpython/Doc/library/time.rst:149:
WARNING: inconsistent term references in translated message. original:
['Availability'], translated: [] [i18n.inconsistent_references]

ken71301 commented:
/home/runner/work/python-docs-zh-tw/cpython/Doc/library/time.rst:149:
WARNING: inconsistent term references in translated message. original:
['Availability'], translated: [] [i18n.inconsistent_references]
通靈了這個，他原文不是 :ref:`Availability <availability>`
只有 Availability

mattwang44 commented:
強！我上次通靈失敗直接[放棄翻譯](https://github.com/python/python-docs-zh-tw/pull/984/commits/d95eb266db4cc3325a00df0b8b2d226cc0d6338b)XD

mattwang44 reviewed (approved):
LGTM!
